VMEncryption/main/DiskUtil.py

```diff
@@ -410,6 +510,33 @@ def _extract_luks_version_from_dump(self, luks_dump_out):
             if "version:" in line.lower():
                 return line.split()[-1]
 
+    def _extract_luksv2_token(self, luks_dump_out):
+        """
+        ...
+        Tokens:
+            1: Azure_Disk_Encryption_BackUp
```

Comment:
Shouldn't the token Id of `Azure_Disk_Encryption_BackUp` be 6? Or Am I missing something?

Reply:
yes, we keeping it into 6, but not relay on token_id. description is meant for specifying Tokens: in LUKS2 header.